packages/jaeger-ui/src/components/common/copy-icon.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react';
+
+import { Icon, Tooltip } from 'antd';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+type propsType = {
+  copyText: string,
+  tooltipTitle: string,
+};
+
+type stateType = {
```

Comment:
Can you use initial caps on type names? E.g. `StateType`

Reply:
capitalized